.github/workflows/ci.yml

```diff
@@ -113,3 +161,16 @@ jobs:
           draft: true
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Upload output artifacts
+        uses: actions/upload-artifact@v3
```

Comment:
:art: this appears to run unconditionally - I think we should scope this to just the `ubuntu` runs within this fork

Reply:
I wanted it to run unconditionally (thats why I had it upload the .exe, .app, .msi, .zip) artifacts from windows/macos. 

I didn't realize until just now that you have it so that only linux is actually packaged https://github.com/shiftkey/desktop/blob/1d3d1567d8aed7d31de99acc1dc92a8bbdcdfeb1/.github/workflows/ci.yml#L145-L147 so that explains why they never uploaded anything 😆

Reply:
solved in latest commits